test-retry-runner.ps1

```diff
@@ -9,21 +9,22 @@ $ErrorActionPreference = "Stop"
 
 if ($IsWindows) {
     $projectsToSkip = @(
-        "Microsoft.DotNet.Interactive.CSharpProject.Tests"
+        "Microsoft.DotNet.Interactive.CSharpProject.Tests",
+        "Microsoft.DotNet.Interactive.Jupyter.Tests"
```

Comment:
Is it intentional that Jupyter tests are skipped entirely on windows - but only partially otherwise?

Reply:
No, that was an experiment. Good catch!